### PR TITLE
Clear Cache button and Confirmation Dialog

### PR DESCRIPTION
**Pull request by NikhilNarayana**

### CODE REVIEW

File: src/dolphin/manager.ts [hunk 1]
```diff
@@ -108,6 +110,16 @@ export class DolphinManager extends EventEmitter {
...
     downloadAndInstallDolphin(launchType, log.info, true);
   }
+
+  public async clearCache(launchType: DolphinLaunchType) {
+    const userFolder = await findUserFolder(launchType);
+    const cacheFolder = path.join(userFolder, "Cache");
+    try {
+      fs.remove(cacheFolder);
+    } catch (err) {
+      log.error(err);
```
Comment (vinceau) [inline src/dolphin/manager.ts:120]: Should we not throw a proper error here so the frontend can get the error?

File: src/renderer/components/ConfirmationModal.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    onSubmit();
+    onClose();
+  };
+
+  return (
+    <Dialog open={open} onClose={onClose} fullWidth={true} fullScreen={fullScreen} disableBackdropClick={true}>
+      <form onSubmit={handleSubmit}>
+        <StyledDialogTitle>{title}</StyledDialogTitle>
+        <DialogContent style={{ display: "flex" }}>
+          <div>{content}</div>
```
Comment (vinceau) [inline src/renderer/components/ConfirmationModal.tsx:35]:
```suggestion
          <div>{content}</div>
          {children}
```
I think just remove the `content` prop and allow for children to be rendered.

**Review by vinceau — CHANGES_REQUESTED** [review]

File: src/renderer/components/ConfirmationModal.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    console.log("submitting form...");
+    onSubmit();
+    onClose();
+  };
+
+  return (
+    <Dialog open={open} onClose={onClose} fullWidth={true} fullScreen={fullScreen} disableBackdropClick={true}>
+      <form onSubmit={handleSubmit}>
+        <StyledDialogTitle>{title}</StyledDialogTitle>
+        <DialogContent style={{ display: "flex" }}>{children}</DialogContent>
```
Comment (vinceau) [inline src/renderer/components/ConfirmationModal.tsx:33]: Why is this `display: flex` here?

**Review by vinceau — CHANGES_REQUESTED** [review]

File: src/renderer/components/ConfirmationModal.tsx (hunk 3 above)
Comment (NikhilNarayana) [reply src/renderer/components/ConfirmationModal.tsx:33]: I copied it from StartBroadcastDialog

**Review by vinceau — APPROVED** [review]

**Pull request merged**